package/MDAnalysis/core/groups.py

```diff
@@ -1487,12 +1471,46 @@ class AtomGroup(GroupBase):
     AtomGroup instances are always bound to a
     :class:`MDAnalysis.core.universe.Universe`. They cannot exist in isolation.
 
+
+    .. rubric:: Deprecated functionality
+
+    *Instant selectors* will be removed in the 1.0 release.  See issue `#1377
+    <https://github.com/MDAnalysis/mdanalysis/issues/1377>`_ for more details.
+
+    Atoms can also be accessed in a Pythonic fashion by using the atom name as
```

Comment:
If we're deprecating something, can we remove the documentation for it?  Someone might skip the dep warning, then read about this great feature they want to use forevermore....